src/partials/head.hbs

```diff
@@ -12,7 +12,8 @@
       const banner = document.getElementById(bannerId);
       if (!banner) return;
 
-      const hasSeenBanner = window.sessionStorage.getItem(sessionKey) || false;
+      const storageKey = sessionKey;
```

Comment:
We might need unique keys for different announcement types. For example, if both site-wide and page-wide is enabled, and I close site-wide, page-wide should stay open